src/shared/components/post/create-post.tsx

```diff
@@ -166,10 +196,25 @@ export class CreatePost extends Component<
 
   render() {
     const { selectedCommunityChoice, siteRes, loading } = this.state;
-
-    const locationState = this.props.history.location.state as
```

Comment:
Removing this broke the cross-post button in `src/shared/components/common/content-actions/cross-post-button.tsx`

I suppose it needs to merge these two objects.

Reply:
I got it now.